config/default.yml

```diff
@@ -125,3 +125,12 @@ RSpec/NamedSubject:
 RSpec/SubjectStub:
   Description: 'Checks for stubbed test subjects'
   Enabled: true
+
+RSpec/Predicate:
+  Description: 'Prefer using predicate matcher instead of truthiness'
```

Comment:
Can you update this to simply convey that it enforces one of several styles. Checkout the descriptions of the other `EnforcedStyle` cops for examples of what I mean